src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -597,6 +625,58 @@ class CombatLogParser extends MainCombatLogParser {
           </dfn>
         ),
       },
+      this.modules.archiveoffaith.active && {
+        id: ITEMS.ARCHIVE_OF_FAITH.id,
+        icon: <ItemIcon id={ITEMS.ARCHIVE_OF_FAITH.id} />,
+        title: <ItemLink id={ITEMS.ARCHIVE_OF_FAITH.id} />,
+        result: (
+          <span>
+            <dfn data-tip="The actual effective healing contributed by the Archive of Faith on-use effect.">
+              {((AOFHealing * 100) || 0).toFixed(2)} % / {formatNumber(this.modules.archiveoffaith.healing / fightDuration * 1000)} HPS
+            </dfn>
+            {' '}
+            (HoT: <dfn data-tip="The actual effective healing contributed by the HOT from the full channel of the Archive of Faith on-use effect.">
+              {((AOFHOTHealing * 100) || 0).toFixed(2)} % / {formatNumber(this.modules.archiveoffaith.healingHOT / fightDuration * 1000)} HPS
+            </dfn>)
+          </span>
+        ),
+      },
+      this.modules.barbaricMindslaver.active && {
+        id: ITEMS.BARBARIC_MINDSLAVER.id,
+        icon: <ItemIcon id={ITEMS.BARBARIC_MINDSLAVER.id} />,
+      title: <ItemLink id={ITEMS.BARBARIC_MINDSLAVER.id} />,
+        result: (
+          <dfn data-tip="The actual effective healing contributed by the Barbaric Mindslaver equip effect.">
+            {((barbaricMindslaverHealingPercentage * 100) || 0).toFixed(2)} % / {formatNumber(this.modules.barbaricMindslaver.healing / fightDuration * 1000)} HPS
+          </dfn>
+        ),
+      },
+      this.modules.seaStar.active && {
+        id: ITEMS.SEA_STAR_OF_THE_DEPTHMOTHER.id,
+        icon: <ItemIcon id={ITEMS.SEA_STAR_OF_THE_DEPTHMOTHER.id} />,
+      title: <ItemLink id={ITEMS.SEA_STAR_OF_THE_DEPTHMOTHER.id} />,
+        result: (
+          <dfn data-tip="The actual effective healing contributed by the Sea Star of the Depthmother equip effect.">
+            {((seaStarHealingPercentage * 100) || 0).toFixed(2)} % / {formatNumber(this.modules.seaStar.healing / fightDuration * 1000)} HPS
+          </dfn>
+        ),
+      },
+      this.modules.dgd.active && {
+        id: ITEMS.DECEIVERS_GRAND_DESIGN.id,
+        icon: <ItemIcon id={ITEMS.DECEIVERS_GRAND_DESIGN.id} />,
+        title: <ItemLink id={ITEMS.DECEIVERS_GRAND_DESIGN.id} />,
+        result: (
+          <span>
+            <dfn data-tip="The actual effective healing contributed by the Deciever's Grand Design on-use effect.">
```

Comment:
Would it be sufficient to show the total healing here and the breakdown (dot and shield proc) in the tooltip instead?